dpnp/dpnp_iface_manipulation.py

```diff
@@ -2229,6 +2229,14 @@ def unique(
     This is done by making the specified axis the first dimension of the array
     (move the axis to the first dimension to keep the order of the other axes)
     and then flattening the subarrays in C order.
+    If NaN values are in the input array, a single NaN is put to the end of the
```

Comment:
Is it true? Don't we control it with `equal_nan`? Looking at NumPy, `equal_nan` was introduced in 1.24 and such a note that was added in NumPy 1.21 probably should have been removed.

Reply:
Yes, you are right, I'll remove that.